modules/apps/dynamic-data-mapping/dynamic-data-mapping-form-taglib/src/test/java/com/liferay/dynamic/data/mapping/form/taglib/servlet/taglib/DDMFormRendererTagTest.java

```diff
@@ -353,7 +358,9 @@ protected void setUpDDMFormInstanceRecordLocalService() throws Exception {
 			_ddmFormInstanceRecord
 		);
 
-		field.set(_ddmFormTaglibUtil, _ddmFormInstanceRecordLocalService);
+		field.set(
```

Comment:
@slnn Please mock `*ServiceUtil.getService()` method instead of using reflection.